javascript/exosecurity/features/support/steps-given.js

```diff
@@ -0,0 +1,35 @@
+const { defineSupportCode } = require('cucumber')
```

Comment:
`Given` can be required directly from cucumber. Same with When / Then. Lets use that instead of `defineSupportCode`